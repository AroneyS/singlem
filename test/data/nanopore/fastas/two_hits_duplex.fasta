>two_hits_duplex
ATGAATATCCGGCAGTTTTTAACCGAGATCGGGAACCATATCCAGGATATTAACAGTAGCTGAAGTTACTGACTTACGTTCACAATTACGTGAAGCTGGTGTTGAGTATAAAGTATACAAAAACACTATGGTACGTCGTGCAGCTGAATGAAATGGCAGGTTTAAAGTTCAATAGAATCTATGATGGACTCAAGGACCTCTACAGGGTCAGGGGCTGAAATGGCAGGTTTAAAGTTCAATAGAATCTATGATGGACTCAAGGACCTCTACAGGGTCAGGGGCTGAAATGGCAGGTTTAAAGTTCAATAGAATCTATGATGGACTCAAGGACCTCTACAGGGTCAGGGGCTGAAATGGCAGGTTTAAAGTTCAATAGAATCTATGATGGACTCAAGGACCTCTACAGGGTCAGGGGCGATGGCGGTAAAGCCACTCCCGGCCCACCATTAGGTCCAGCAATCGGACCCCTAGGTATCATGAATATCCGGCAGTTTTTAACCGAGATCGGGAACCATATCCAGGATATCCTGGATATGGTTCCCGATCTCGGTTAAAAACTGCCGGATATTCATGATACCTAGGGGTCCGATTGCTGGACCTAATGGTGGGCCGGGAGTGGCTTTACCGCCATCGCCCCTGACCCTGTAGAGGTCCTTGAGTCCATCATAGATTCTATTGAACTTTAAACCTGCCATTTCAGCCCCTGACCCTGTAGAGGTCCTTGAGTCCATCATAGATTCTATTGAACTTTAAACCTGCCATTTCAGCCCCTGACCCTGTAGAGGTCCTTGAGTCCATCATAGATTCTATTGAACTTTAAACCTGCCATTTCAGCCCCTGACCCTGTAGAGGTCCTTGAGTCCATCATAGATTCTATTGAACTTTAAACCTGCCATTTCATTCAGCTGCACGACGTACCATAGTGTTTTTGTATACTTTATACTCAACACCAGCTTCACGTAATTGTGAACGTAAGTCAGTAACTTCAGCTACTGTTAATATCCTGGATATGGTTCCCGATCTCGGTTAAAAACTGCCGGATATTCAT
