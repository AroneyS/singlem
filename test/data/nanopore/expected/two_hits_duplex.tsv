gene	sample	sequence	num_hits	coverage	taxonomy
4.12.22seqs	two_hits_duplex	GATGGCGGTAAAGCCACTCCCGGCCCACCATTAGGTCCAGCAATCGGACCCCTAGGTATC	1	1.06	
4.11.22seqs	two_hits_duplex	TTACGTTCACAATTACGTGAAGCTGGTGTTGAGTATAAAGTATACAAAAACACTATGGTA	1	1.06	
